Formula/fd.rb

```diff
@@ -14,8 +14,13 @@ class Fd < Formula
   depends_on "rust" => :build
 
   def install
+    ENV.store "SHELL_COMPLETIONS_DIR", buildpath/"target/release"
```

Comment:
Please change this to `ENV["SHELL_COMPLETIONS_DIR"] = buildpath/"target/release"`.

Reply:
Done.